.env

```diff
@@ -1,13 +1,11 @@
 REACT_APP_REST_BASE="/${REACT_APP_CONFIG_ENV}"
 
-REACT_APP_MGNL_SUB_ID="YOUR_SUBSCRIPTION_ID"
+REACT_APP_MGNL_SUB_ID="vstinte7ggsb78qm"
```

Comment:
@czimmermann is this intended? - replacing the ID

Reply:
Thanks! No it was not intended. Added another commit to fix it.